src/content/docs/dropins/all/extending.mdx

```diff
@@ -46,4 +47,152 @@ Component that provides placeholders to add other components. You can use a drop
 
 Overloaded term in web development. Everything is a component. It's components all the way down. This is why we need to be specific about what kind of component we are talking about. For example, from top-to-bottom, big-to-small, a **drop-in component** can contain multiple **container components** that can contain multiple **slot components** that can contain multiple **UI components**.
 
-</Vocabulary>
\ No newline at end of file
+</Vocabulary>
+
+## Add third-party components
+
+This tutorial describes the steps needed to integrate a third-party component (the Yotpo ratings & reviews component) within the product detail drop-in. We will add one component instance below the product's title (under the title slot), and a second instance below the product gallery.
```

Comment:
I would be more clear with the Yotpo integration being just an example. There are multiple 3rd party tools that the client can use for the ratings, and we don't want them to assume that they are required to use the same, or that we specifically recommend it. 
It's simply the case that Bulk happens to use the Yotpo API.

Reply:
+1 to what Daniela said. I would even go further and completely abstract away from Yotpo and add a generic example like `api.rating.service.com`. Otherwise we need to keep the docs up to data with whatever changes Yotpo come up with :)